### PR TITLE
Kills grown/misc.dm

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

grown/misc.dm is NO MORE!

## Why It's Good For The Game

miscellaneous files are notoriously good at hiding where things are, and files are 100% free. 

image

## Changelog
:cl:
code: killed the grown misc file!
/:cl:

### CODE REVIEW

**Comment by Colovorat** [comment]
image
you really want to kill them aren't you?

File: code/game/objects/structures/fluff.dm [hunk 1]
```diff
@@ -262,34 +262,6 @@
...
-			to_chat(user,span_notice("You finish trimming \the [src]."))
-			opacity = FALSE
-	else
-		return ..()
-/**
- * useful for mazes and such
- */
-/obj/structure/fluff/hedge/opaque
-	opacity = TRUE
-
```
Comment (interestingusernam3) [inline code/game/objects/structures/fluff.dm:292]: Change not mentioned in PR body

**Comment by necromanceranne** [comment]
arm, I think at least one green name is going to blow a gasket for your pr names being nondescriptive.

File: code/game/objects/structures/fluff.dm (hunk 1 above)
Comment (tralezab) [reply code/game/objects/structures/fluff.dm:292]: Fluff.dm ironically enough is just a thinly veiled miscellaneous file, i have moved them to where they will be found

**Comment by MrDoomBringer** [comment]
> arm, I think at least one green name is going to blow a gasket over your pr names being nondescriptive.

now you understand

**Comment by tralezab** [comment]
> arm, I think at least one green name is going to blow a gasket over your pr names being nondescriptive.

point incredibly proven with one weird trick

**Pull request merged**